### ISSUE TITLE
Merge root template default timeout is not enough

### ISSUE THREAD

**Issue by youyk** [zstackorg/zstack#204]
Currently the default timeout for merge root template is not enough (300s). If the vm root volume is large, the merge will be timeout:

```
{
  "success": false,
  "apiId": "24908985502d45c59fe6993ce02a713e",
  "headers": {
    "schema": {}
  },
  "session": {
    "callid": "api-FJfQaUUq",
    "uuid": "9554a575d3444446bbd6deb23bbf2959"
  },
  "error": {
    "code": "SYS.1006",
    "cause": {
      "code": "SYS.1006",
      "cause": {
        "code": "SYS.1001",
        "description": "A message or a operation timeout",
        "details": "[Async Http Timeout] url: http://192.168.200.2:7070/localstorage/volume/createtemplate, timeout after 300[SECONDS], command: {\"installPath\":\"/zstack_ps/templateWorkspace/image-cdab68f2822b4efbb7c5368f41ff45d9/cdab68f2822b4efbb7c5368f41ff45d9.qcow2\",\"volumePath\":\"/zstack_ps/rootVolumes/acct-36c27e8ff05c4780bf6d2fa65700f22e/vol-0a9e809eac0e4ec593ec900ac0393e4d/0a9e809eac0e4ec593ec900ac0393e4d.qcow2\"}"
      },
      "description": "An operation failed",
      "details": "failed to create template from root volume[uuid:0a9e809eac0e4ec593ec900ac0393e4d] on primary storage[uuid:631d2407d5864798a2b27699449ecc29]"
    },
    "description": "An operation failed",
    "details": "failed to create image from root volume[uuid:0a9e809eac0e4ec593ec900ac0393e4d] on all backup storage, see cause for one of errors"
  },
  "id": "f200e4d03d2942ea8c6373ede5ff8077",
  "creatingTime": 1453966168079
}
```

**Comment by youyk** [comment]
manually do the command for a 40G root volume, it cost more than 2mins. If there are other disk operation in the system, the time will be increased.

time /usr/bin/qemu-img convert -f qcow2 -O qcow2 /zstack_ps/rootVolumes/acct-36c27e8ff05c4780bf6d2fa65700f22e/vol-0a9e809eac0e4ec593ec900ac0393e4d/0a9e809eac0e4ec593ec900ac0393e4d.qcow2 /zstack_ps/need_remove.qcow2

real    2m15.777s
user    0m2.288s
sys     0m45.140s

**Comment by zxwing** [comment]
fixed in ee8715ddaaa5eac122cd2e7ce6aaf88fdeb29cff
